community/front-end/ofe/infrastructure_files/gcs_bucket/clusters/ansible_setup/roles/spack_setup/tasks/main.yaml

```diff
@@ -13,10 +13,13 @@
 # limitations under the License.
 
 ---
-- name: Spack config externals
-  ansible.builtin.shell: |
-    module load openmpi
-    {{ spack_dir }}/bin/spack external find --scope=system --not-buildable
+- name: Load openmpi module
+  ansible.builtin.shell: ". /usr/share/Modules/init/sh;module load openmpi"
```

Comment:
This would be cleaner as a multiline string like it was before:

```yaml
ansible.builtin.shell: |
  . /usr/share/Modules/init/sh
  module load openmpi
```